esx_service/utils/auth.py

```diff
@@ -91,34 +91,34 @@ def get_default_tenant():
         return None, None, None
 
 
-def get_default_privileges():
-    """ Get DEFAULT privileges by querying the auth DB.
-        DEFAULT privilege will match any (tenant_uuid, datastore) pair which
-        does not have an entry in privileges table explicitly
+def get_all_ds_privileges(tenant_uuid):
+    """ Get privileges on _ALL_DS by querying the auth DB.
+        For the tenant with given tenant_uuid, this privilege
+        will match any datastore which does not have an entry
+        in privileges table explicitly
```

Comment:
@govint "volume create x@DS2 from VMG1 will check the privilege <VMG1, ALL_DS> to see if create is allowed or not.  However, "volume create x@DS1 from VMG2" is not allowed since no privilege is available for VMG2 on DS1. 
So I think  vmgroup isolation still works in this case.